src/main/java/net/countercraft/movecraft/worldguard/listener/ExplosionListener.java

```diff
@@ -1,25 +1,40 @@
 package net.countercraft.movecraft.worldguard.listener;
 
 import com.sk89q.worldguard.protection.flags.Flags;
+import net.countercraft.movecraft.MovecraftLocation;
+import net.countercraft.movecraft.craft.CraftManager;
 import net.countercraft.movecraft.events.ExplosionEvent;
+import net.countercraft.movecraft.worldguard.CustomFlags;
 import net.countercraft.movecraft.worldguard.MovecraftWorldGuard;
+import net.countercraft.movecraft.worldguard.utils.WorldGuardUtils;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
+import org.bukkit.Location;
 
 public class ExplosionListener implements Listener {
     @EventHandler
     public void onExplosion(ExplosionEvent e) {
-        switch (MovecraftWorldGuard.getInstance().getWGUtils().getState(
-                null, e.getExplosionLocation(), Flags.OTHER_EXPLOSION)) {
-            case ALLOW:
-            case NONE:
-                break; // Other-explosion is allowed
+        var wgUtils = MovecraftWorldGuard.getInstance().getWGUtils();
+        Location explosionLocation = e.getExplosionLocation();
+
+        switch (wgUtils.getState(null, explosionLocation, Flags.OTHER_EXPLOSION)) {
             case DENY:
                 // Other-explosion is not allowed
                 e.setCancelled(true);
                 return;
-            default:
+            case ALLOW: // Other-explosion is allowed
+            case NONE:
                 break;
         }
```

Comment:
This is a change in behavior, please revert it to the previous format.

Reply:
it is the same thing i only moved the break cases down, default wasn't needed as all the 3 cases were fulfilled but sure